meeting-notes/2019/Q1/2019-01-28--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,209 @@
+- Blocked: N/A
+- Next:
+  - Git Merge and FOSDEM
+  - Merge scripts together
+  - Improve prototype to generate markdown
+
+@hugomrdias
+- Done:
+  - https://github.com/ipfs/js-ipfs/pull/1830 reviewed feat: load IPLD formats lazily from IPFS 
+  - https://github.com/ipfs/js-ipfs/issues/1827  Bootstrap broken in 0.34.0?
+  - https://github.com/hugomrdias/iso-url/releases/tag/v0.4.5 released iso-url
+  - https://github.com/ipfs/js-ipfs/pull/1839 fix: add cors support for preload-mock-server and update aegir
+  - Debugged a Invalid URL error in the preload tests of js-ifps seems to be related to ipv6 url without brackets and not iso-url related 
+  - Make pullstream-to-stream return a proper readable-stream
+  - Add cov and custom browsers to karma config in aegir (ci prototypes related)
+  - sick thrusday and friday
+- Blocked:
+  - mplex
+- Next:
+  - FOSDEM
+  - Vacation on tuesday
+  - finish pullstream-to-stream
+  - proper-lockfile with extra check for staleness 
+
+
+@mcollina
+- done:
+  - https://github.com/ipfs/js-ipfs/issues/1337#issuecomment-457563157 
+- blocked:
+- next:
+  - analysis of https://github.com/ipfs/benchmarks/issues/203#issuecomment-457246301
+
+
+@achingbrain
+- Done:
+ - npm-registry-client => npm-ipfs-client
+ - Reviewing js-ipld api changes
+- Blocked:
+ - n/a
+- Next:
+ - npm-registry-client => npm-ipfs-client
+ - Merge js-ipld API update PRs
+ - Refactor unixfs/mfs to use async interators
+
+
+@zcstarr
+- Done:
+	- js-ipfs repo async/await
+- Blocked:
+- Next:
+	- js-ipfs repo async/await PR
+
+@elexy
+- done: 
+- blocked:
+  - https://github.com/ipfs/benchmarks/issues/150
+- next: 
+  - https://github.com/ipfs/benchmarks/issues/198
+  - https://github.com/ipfs/benchmarks/issues/169
+  - https://github.com/ipfs/benchmarks/issues/165
+  - https://github.com/ipfs/benchmarks/issues/147
+  - https://github.com/ipfs/benchmarks/issues/35
+
+@litzenberegr
+- done: 
+  - [Added Js - Go tcp](https://github.com/ipfs/benchmarks/pull/213)
+  - [Added Go -> JS Cat tests](https://github.com/ipfs/benchmarks/pull/208)
+  - [Fix for using Large files in peer testing](https://github.com/ipfs/benchmarks/pull/207)
+  - [Added Go -> JS tcp](https://github.com/ipfs/benchmarks/pull/206)
+  - [Added Web Sockets tests for Go ->JS and JS-GO cat tests](https://github.com/ipfs/benchmarks/pull/206)
+- blocked:
+- next:
+  - Add IPFS options per test - https://github.com/ipfs/benchmarks/issues/216
+  - MFS test 
+  - Testing changes
+  - Other tests
+
+@lidel
+- Done:
+	- Quick update on Brave/Chromium: ipfs-companion extension ID is now whitelisted in Brave and we confirmed `chrome.sockets` API can be used to start HTTP server: [ipfs-companion/issues/664#comment](https://github.com/ipfs-shipyard/ipfs-companion/issues/664#issuecomment-458213687) (this is huge, as it confirms js-ipfs-in-webextension as a provider of HTTP Gateway is feasible \o/)
```

Comment:
The world is completely missing on amazing this is. Let's write a script for a fun demoing while at FOSDEM that then we can record :)

Reply:
//cc @lidel